ReactNative/js/screens/Home/components/News.jsx

```diff
@@ -57,15 +57,16 @@ const useNews = newsModule => {
   return data;
 };
 
-const HiddableImage = ({ style, url }) => {
-  const [isHidden, setHidden] = useState(false, [url]);
+const HiddableImage = (props) => {
+  const { style, source } = props;
+  const [isHidden, setHidden] = useState(false, [source]);
   const hide = useCallback(() => setHidden(true), [setHidden]);
   const hiddenStyle = useMemo(
     () => (isHidden ? { height: 0, marginBottom: 0 } : null),
     [isHidden],
   );
   return (
-    <Image style={[style, hiddenStyle]} source={{ uri: url }} onError={hide} />
+    <Image {...props} style={[style, hiddenStyle]} source={{ uri: source }} onError={hide} />
```

Comment:
```suggestion
    <Image style={[style, hiddenStyle]} source={{ uri: source }} onError={hide} />
```
no need to spread the props as all props are already passed

Reply:
```TouchableWithoutFeedback``` provide implicit props which should be pass
From documentation 
```
function MyComponent(props) {
  return (
    <View {...props} style={{flex: 1, backgroundColor: '#fff'}}>
      <Text>My Component</Text>
    </View>
  );
}

<TouchableWithoutFeedback onPress={() => alert('Pressed!')}>
  <MyComponent />
</TouchableWithoutFeedback>;
```
note plz how they spread props 
https://facebook.github.io/react-native/docs/touchablewithoutfeedback